AUTHORS

```diff
@@ -735,5 +735,14 @@ Alex Yursha <alexyursha@gmail.com>
 Steven Vercruysse <stcruy@users.noreply.github.com>
 Aleksanteri Negru-Vode <aleksanteri.negru-vode@aalto.fi>
 Mathieu Darse <mattdarse@gmail.com>
+Connor Peet <connor@peet.io>
+Mayhem <stepien.nicolas@gmail.com>
+Olov Lassus <olov.lassus@gmail.com>
+Phillip Lamplugh <phillip.lamplugh@gmail.com>
+Kohei TAKATA <kt.koheitakata@gmail.com>
+Giovanny Andres Gongora Granada <gioyik@gmail.com>
+Jeffrey Jagoda <jeffrey.jagoda@gmail.com>
+Kelsey <ifoundthemeaningoflife@gmail.com>
```

Comment:
yes, Breseman, @Frijol: your git user.name is just "Kelsey", do you want us to update it on our end to be "Kelsey Breseman" so it shows that way in the AUTHORS doc and commit log?